src/containers/pages/IRSLite/MapLite/helpers.ts

```diff
@@ -0,0 +1,306 @@
+import GeojsonExtent from '@mapbox/geojson-extent';
```

Comment:
Most of the code here is duplicated from [IRS map helpers](https://github.com/onaio/reveal-frontend/blob/macepa-lite-tf/src/containers/pages/IRS/Map/helpers.ts) why not import?

Reply:
@justinlewis this is not yet addressed.